### PR TITLE
[11.0] web_responsive : Support sticky table headers on chrome

### PR DESCRIPTION
**Pull request by grindtildeath**
FF 62.0.3 (64-bit) on ubuntu 16.04 before :
ff_before

FF 62.0.3 (64-bit) on ubuntu 16.04 after :
ff_after

Chrome 70.0.3538.67 (64-bit) on ubuntu 16.04 before :
chrome_before

Chrome 70.0.3538.67 (64-bit) on ubuntu 16.04 after :
chrome_after

### CODE REVIEW

**Comment by simahawk** [comment]
tested on latest chrome on windows too ;)
@grindtildeath FTR you could mention here caniuse.com link and stackoverflow workaround

File: web_responsive/static/src/less/form_view.less [hunk 1]
```diff
@@ -57,13 +57,16 @@
     >div {
         >.table-responsive {
             >.o_list_view {
-                thead {
+                thead tr:nth-child(1) th {
                     position: sticky;
                     top: 0;
+                    background-color: #EEEEEE;
```
Comment (simahawk) [inline web_responsive/static/src/less/form_view.less:63]: isn't there any variable for this color?

File: web_responsive/__manifest__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "name": "Web Responsive",
     "summary": "It provides a mobile compliant interface for Odoo Community "
                "web",
-    "version": "11.0.2.0.0",
+    "version": "11.0.2.0.1",
     "category": "Website",
     "website": "https://laslabs.com/",
```
Comment (simahawk) [inline web_responsive/__manifest__.py]: not related to this change, still: https://github.com/OCA/web

**Review by simahawk — CHANGES_REQUESTED** [review]

File: web_responsive/static/src/less/form_view.less (hunk 1 above)
Comment (grindtildeath) [reply web_responsive/static/src/less/form_view.less:63]: What do you mean ?

Comment (simahawk) [reply web_responsive/static/src/less/form_view.less:63]: `@odoo-color-for-thead` or similar -> likely there's a variable to reuse and you don't hardcode this color

Comment (grindtildeath) [reply web_responsive/static/src/less/form_view.less:63]: https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/less/list_view_extra.less#L4 :man_shrugging:

Comment (simahawk) [reply web_responsive/static/src/less/form_view.less:63]: amazing :rofl:

**Review by simahawk — APPROVED** [review]

**Review by tarteo — CHANGES_REQUESTED** [review]
Is it possible to keep the border on the table header? Now it's all one color gray.

**Comment by grindtildeath** [comment]
@tarteo I tried, but it unfortunately seems not possible and that's why I had to set it to 0.
The border would appear as long as you didn't scroll down yet, but when scrolling the border will disappear and there will be a 1px transparent border between the sticky header cells.
From what I tested, to have the border, we would have to switch `border-collapse` property which is set to `collapse` for `table` on bootstrap (https://github.com/odoo/odoo/blob/11.0/addons/web/static/lib/bootstrap/less/normalize.less#L417), but I couldn't get a good rendering.
Therefore, I dropped the border as IMO usability (not having to scroll up in a big list to know what you're looking at) is more important than esthetics.
Maybe there's another solution to keep it, but I couldn't find any.

**Review by tarteo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@Yajo check this for the v12 migration

**Pull request merged**